sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -238,22 +239,24 @@ func (e *containerEndpoint) String() string {
 func NewManager(
 	clusterEntities EntityStore,
 	externalSrcs externalsrcs.Store,
+	collectorRuntimeConfig collectorruntimeconfig.Store,
 	policyDetector detector.Detector,
 	pubSub *internalmessage.MessageSubscriber,
 ) Manager {
 	enricherTicker := time.NewTicker(tickerTime)
 	mgr := &networkFlowManager{
-		connectionsByHost: make(map[string]*hostConnections),
-		clusterEntities:   clusterEntities,
-		publicIPs:         newPublicIPsManager(),
-		externalSrcs:      externalSrcs,
-		policyDetector:    policyDetector,
-		enricherTicker:    enricherTicker,
-		initialSync:       &atomic.Bool{},
-		activeConnections: make(map[connection]*networkConnIndicator),
-		activeEndpoints:   make(map[containerEndpoint]*containerEndpointIndicator),
-		stopper:           concurrency.NewStopper(),
-		pubSub:            pubSub,
+		connectionsByHost:      make(map[string]*hostConnections),
+		clusterEntities:        clusterEntities,
+		publicIPs:              newPublicIPsManager(),
+		externalSrcs:           externalSrcs,
+		collectorRuntimeConfig: collectorRuntimeConfig,
```

Comment:
nit rename to `externalSrcsStore`, `clusterEntitiesStore` and `collectorRuntimeConfigStore`

Reply:
Done